sdk/eventhub/Azure.Messaging.EventHubs/tests/Producer/EventHubProducerClientTests.cs

```diff
@@ -665,6 +1185,18 @@ public async Task CloseAsyncDoesNotCloseTheConnectionWhenNotOwned()
                     .GetProperty("RetryPolicy", BindingFlags.Instance | BindingFlags.NonPublic)
                     .GetValue(producer);
 
+        /// <summary>
+        ///   Sets <see cref="EventHubProducerClient.IsClosed"/> property using its protected accessor.
+        /// </summary>
+        ///
+        /// <param name="isClosed">The value for the<see cref="EventHubProducerClient.IsClosed"/> property.</param>
+        ///
+        private static void SetIsClosed(EventHubProducerClient producer, bool isClosed) =>
```

Comment:
nit: I'm probably being annoying with the nitpicking at this point, but I think this is the last one. Could you separate the parameters in two lines?